### PR TITLE
add contributor jordi_248

### PR DESCRIPTION
**Pull request by Jordan2204**
Hello, I'm a new contributor jordi_248 and I will be happy to be a member of this community.

### CODE REVIEW

File: content/authors/jordi_248/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: "Jordi_248"
+images: ["image.png"]
```
Comment (RMPR) [inline content/authors/jordi_248/_index.md:3]: Can you give a more meaningful name to your image file?

File: content/post/contributors/jordi248.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Jordi_248"
+date: 2022-03-12T20:13:26+01:00
+images: []
+authors: [Jordi_248]
+---
+
+Hello everyone, my name is ANAFACK Diderot Jordan (Jordi_248). I am a software engineer. I am a computer enthusiast in general but I am more specialized in web development in which I already have 4 years of experience and more recently I started the mobile development with flutter. I am comfortable with the following languages: php, python, javascript, java, c. I am as comfortable with backend technologies as front-end technologies.
```
Comment (RMPR) [inline content/post/contributors/jordi248.md:8]:
```suggestion
Hello everyone, my name is ANAFACK Diderot Jordan (Jordi_248). I am a software engineer. I am a computer enthusiast in general but I am more specialized in web development in which I already have 4 years of experience and more recently I started the mobile development with flutter. I am comfortable with the following languages: php, python, javascript, java, c. I am as comfortable with back-end technologies as front-end technologies.
```

**Review by RMPR — COMMENTED** [review]
Thanks for your PR.

**Review by elhmn — APPROVED** [review]
Thank you for your contribution, and welcome to the community. We hope to see more of contributions in the future 🥳

**Pull request merged**